src/pages/EmployeeDetail/components/EmployeeForm.tsx

```diff
@@ -47,6 +54,30 @@ export default function EmployeeForm(props: EmployeeFormProps) {
         innerRef={formik}
       >
         <Form id="emp-form" noValidate>
+          <div className={"profile-upload " + (props.isView ? "is-view" : "")}>
+            <label htmlFor="imageUpload" tabIndex={0}>
+              <img src={profileImg} alt="employee profile" />
+              <div className={props.isView ? "is-view" : ""}>
```

Comment:
use semantic tags than using DIVs as Div is a heavy element

Reply:
okay